raiden_contracts/contract_manager.py

```diff
@@ -125,6 +126,12 @@ def contracts_deployed_path(
     return data_path.joinpath(f'deployment_{"services_" if services else ""}{chain_name}.json')
 
 
+class DeploymentModule(Enum):
+    RAIDEN = 'raiden'
```

Comment:
you can also have those as: RAIDEN = 1, SERVICES = 2, ALL = 3. No need to be strings.

Reply:
Probably I should go for https://docs.python.org/3/library/enum.html#enum.auto